alerts/cluster-monitoring-operator/TargetDown.md

```diff
@@ -203,19 +241,25 @@ spec:
       app.kubernetes.io/name: node-exporter
       app.kubernetes.io/part-of: openshift-monitoring
 ```
+
 In the `Prometheus` that scrapes this target, the volume mounted at `/etc/prometheus/secrets/*`
 are the secrets having the same name as the subdirectory. In this example, the secret
 is `metrics-client-certs` in the namespace `openshift-monitoring`. Now we extract
 the certificate from the secret using this command:
-```bash
+
+```shell
 oc extract secret/$SECRET_NAME -n $NAMESPACE --keys=tls.crt --to=- > certificate.crt
 ```
+
 Then we inspect is expiration date.
```

Comment:
```suggestion
Then we inspect its expiration date.
```

Reply:
out of scope of the PR (was there before the change), but done.